history/cultures/naga.txt

```diff
@@ -1,39 +1,39 @@
 100.1.1 = {
-	discover_innovation = innovation_motte 
-	discover_innovation = innovation_catapult 
-	discover_innovation = innovation_bannus 
-	discover_innovation = innovation_quilted_armor 
-	discover_innovation = innovation_mustering_grounds 
-	discover_innovation = innovation_barracks 
+	discover_innovation = innovation_motte
+	discover_innovation = innovation_catapult
+	discover_innovation = innovation_bannus
+	discover_innovation = innovation_quilted_armor
+	discover_innovation = innovation_mustering_grounds
+	discover_innovation = innovation_barracks
 	#
-	discover_innovation = innovation_city_planning 
-	discover_innovation = innovation_crop_rotation 
-	discover_innovation = innovation_development_01 
-	discover_innovation = innovation_plenary_assemblies 
-	discover_innovation = innovation_gavelkind 
-	discover_innovation = innovation_casus_belli 
-	discover_innovation = innovation_ledger 
-	discover_innovation = innovation_currency_01 
+	discover_innovation = innovation_city_planning
+	discover_innovation = innovation_crop_rotation
+	discover_innovation = innovation_development_01
+	discover_innovation = innovation_plenary_assemblies
+	discover_innovation = innovation_gavelkind
+	discover_innovation = innovation_casus_belli
+	discover_innovation = innovation_ledger
+	discover_innovation = innovation_currency_01
 	#
 	discover_innovation = innovation_naga_army
 	#
 	join_era = culture_era_early_medieval
 }
 270.1.1 = {
-	discover_innovation = innovation_arched_saddle 
-	discover_innovation = innovation_burhs 
-	discover_innovation = innovation_house_soldiers 
-	discover_innovation = innovation_horseshoes 
-	discover_innovation = innovation_mangonel 
-	discover_innovation = innovation_battlements 
+	discover_innovation = innovation_arched_saddle
+	discover_innovation = innovation_burhs
+	discover_innovation = innovation_house_soldiers
+	discover_innovation = innovation_horseshoes
+	discover_innovation = innovation_mangonel
+	discover_innovation = innovation_battlements
 	#
-	discover_innovation = innovation_chronicle_writing 
-	discover_innovation = innovation_armilary_sphere 
-	discover_innovation = innovation_baliffs 
-	discover_innovation = innovation_development_02 
-	discover_innovation = innovation_manorialism 
-	discover_innovation = innovation_currency_02 
-	discover_innovation = innovation_hereditary_rule 
+	discover_innovation = innovation_chronicle_writing
+	discover_innovation = innovation_armilary_sphere
+	discover_innovation = innovation_baliffs
+	discover_innovation = innovation_development_02
+	discover_innovation = innovation_manorialism
+	discover_innovation = innovation_currency_02
+	discover_innovation = innovation_hereditary_rule
```

Comment:
did you change anything besides the spacing in this file? if not would you mind discarding these changes as to not crowd up the PR?